### PR TITLE
Fix new qemu-kvm version have no rhev issues

### PR DESCRIPTION
**Pull request by kylazhang**
Fix new qemu-kvm version have no rhev issues
Signed-off-by: Kylazhang <weizhan@redhat.com>

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -2062,6 +2062,8 @@ def get_qemu_version(params=None):
             version['update'] = int(search_result.group(3))
         if "rhev" in str(line).lower():
             version['is_rhev'] = True
+        if "mod" in str(line).lower():
```
Comment (chunfuwen) [inline virttest/utils_misc.py]: whole word 'module' is meaningful than  "mod" , change from "mod" to "module" ?

Comment (kylazhang) [reply virttest/utils_misc.py]: no, still have some module named scrmod means scratch module

**Comment by yanqzhan** [comment]
1. A comparison on latest qemu-kvm-4.2.0-19.module+el8.***. It works well.
Before:
]# python3 /root/test-files/test_qemu_version.py 
Installed qemu version is: %s {'major': 4, 'minor': 2, 'update': 0, 'is_rhev': False}
The qemu version is lower than 2.9.0
After:
]# python3 /root/test-files/test_qemu_version.py 
Installed qemu version is: %s {'major': 4, 'minor': 2, 'update': 0, 'is_rhev': False, 'module': True}
The qemu version is not lower than 2.9.0

]# cat /root/test-files/test_qemu_version.py 
from virttest import utils_misc

installed_version = utils_misc.get_qemu_version()
print("Installed qemu version is: %s", installed_version)
if utils_misc.compare_qemu_version(2, 9, 0):
    print("The qemu version is not lower than 2.9.0")
else:
    print("The qemu version is lower than 2.9.0")

Some other comparison can be done a little later.

**Comment by yanqzhan** [comment]
1.2. Installed qemu version is: %s {'major': 4, 'minor': 2, 'update': 0, 'is_rhev': False, 'module': True}
The qemu version is lower than 4.5.0

2. qemu-kvm-rhev-2.12.0-18.el7_*
2.1('Installed qemu version is: %s', {'module': False, 'major': 2, 'update': 0, 'minor': 12, 'is_rhev': True})
The qemu version is not lower than 2.9.0
2.2 ('Installed qemu version is: %s', {'major': 2, 'update': 0, 'minor': 12, 'is_rhev': True})
The qemu version is lower than 2.13.0

3.qemu-kvm-1.5.3-160.el7_*
3.1 ('Installed qemu version is: %s', {'module': False, 'major': 1, 'update': 3, 'minor': 5, 'is_rhev': False})
The qemu version is lower than 2.9.0
3.2 ( 'Installed qemu version is: %s', {'major': 1, 'update': 3, 'minor': 5, 'is_rhev': False})
The qemu version is lower than 1.3.0

Hi @kylazhang , from 3.2 when qemu-kvm on rhel7, compare_qemu_version() always returns False. Is here can be further fixed or ignored? Since it was already like that before this pr, and can be avoid by giving an 'is_rhev=False' to compare_qemu_version().

**Pull request merged**